Formula/libwebsockets.rb

```diff
@@ -13,9 +14,23 @@ class Libwebsockets < Formula
 
   depends_on "cmake" => :build
   depends_on "openssl"
+  depends_on "libev"
+  depends_on "libuv"
+  depends_on "libevent"
 
   def install
-    system "cmake", ".", *std_cmake_args
+    cmake_args = std_cmake_args.concat %w[
+      -DLWS_WITH_LIBEV=ON
+      -DLWS_WITH_LIBUV=ON
+      -DLWS_WITH_LIBEVENT=ON
+      -DLWS_WITH_HTTP2=ON
+      -DLWS_IPV6=ON
+      -DLWS_UNIX_SOCK=ON
+      -DLWS_WITH_PLUGINS=ON
+      -DLWS_WITHOUT_TESTAPPS=ON
+    ]
+
+    system "cmake", ".", *cmake_args
```

Comment:
As a matter of style, just do:

```
  system "cmake", ".", *std_cmake_args,
                  "-DLWS_WITH_LIBEV=ON",
                  …
```

Reply:
Updated as #21653, the old fork is deleted so I can't update this PR anymore.